gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientImpl.java

```diff
@@ -136,6 +146,45 @@ public WritableByteChannel create(StorageResourceId resourceId, CreateObjectOpti
     return channel;
   }
 
+  /**
+   * See {@link GoogleCloudStorage#createBucket(String, CreateBucketOptions)} for details about
+   * expected behavior.
+   */
+  @Override
+  public void createBucket(String bucketName, CreateBucketOptions options) throws IOException {
```

Comment:
is this tested by the integration tests?

Reply:
Yes, https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageTest.java#L168 tests create bucket scenarios. This test is parameterised to use the java-storage client in this PR. https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageIntegrationTest.java#L34